host/host.go

```diff
@@ -0,0 +1,38 @@
+package host
+
+import (
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/taskcluster/shell"
+)
+
+func RunCommand(cmd *exec.Cmd) (combinedOutput string, err error) {
```

Comment:
I think this should take a slice of strings (preventing the need to call this function like `RunCommand(exec.Command([]{"path", "arg1", "arg...", "argn"}))`) for convenience

Reply:
Yeah, maybe. I was torn between the two. The reason I chose to pass in the command, is because it gives the caller the freedom to set env variables or other settings inside the command. Sometimes we set auth tokens, or env vars, in the commands we run, so I figured this was the way to provide the most freedom, but it does come at the cost of making the call site a little bit more clunky.

Reply:
I take this back - it's worth allowing the caller the flexibility to set up an `*exec.Cmd` with things set on it.